MovieHubRx/Utils/LanguageType.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  LanguageType.swift
+//  MovieHubRx
+//
+//  Created by Duy Nguyễn on 22/05/2024.
+//
+
+import Foundation
+import UIKit
+
+enum LanguageType {
```

Comment:
bỏ localized, e dùng gì thì `LanguageType.en.rawValue` như a Đông nói
```suggestion
enum LanguageType: String {
```